gui/LoadThemAllDialog.py

```diff
@@ -71,6 +73,9 @@ def __init__(self, parent, iface):
         self.btnCancel.setVisible(False)
         self.btnCancel.clicked.connect(self.cancelLoad)
 
+        if Qgis.versionInt() < 31800 and has_point_cloud_provider():
```

Comment:
If I understood well, you probably meant 

`if Qgis.versionInt() < 31800 or not has_point_cloud_provider():`

?

(i.e., the tab will be disabled if QGIS version is lesser than 3.18 OR if the QGIS version is greater or equal than 3.18 but it doesn't have Point Cloud providers installed)